include/pros/apix.h

```diff
@@ -52,6 +52,9 @@ extern "C" {
 
 typedef void* queue_t;
 typedef void* sem_t;
+typedef void (*generic_fn_t)(void);
+
+void add_daemon_hook(generic_fn_t hook);
```

Comment:
Doxygen, obviously.

Also, I'm unsold on the names of both `generic_fn_t` and `add_daemon_hook`.

Reply:
Probably `post_processing_hook_fn_t` and `add_post_processing_hook()`